### PR TITLE
Mention Exoterm as an updated urxvt fork with sixel support

### PR DESCRIPTION
**Pull request by tomas**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -225,6 +225,8 @@ Which references VTE upstream: https://gitlab.gnome.org/GNOME/vte/-/issues/253
 
 Currently there is only an old fork (from 2018) that support sixel: https://github.com/saitoha/rxvt-unicode-sixel
```
Comment (smasher164) [inline content/_index.md:226]: Do you mind editing the "Currently there is only" to "There is"?

**Comment by smasher164** [comment]
Thanks for submitting this. Please see my comment.
Do you mind running `hugo -D` after making changes?

File: content/_index.md (hunk 1 above)
Comment (tomas) [reply content/_index.md:226]: Ok, will do later

Comment (tomas) [reply content/_index.md]: Done

**Review by smasher164 — APPROVED** [review]

**Pull request merged**